fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityTypeBuilder.java

```diff
@@ -250,20 +254,26 @@ public FabricEntityTypeBuilder<T> specificSpawnBlocks(Block... blocks) {
 		return this;
 	}
 
+	/**
+	 * Sets the features this entity requires. If the feature is not enabled,
```

Comment:
```suggestion
	 * Sets the features this entity requires. If a feature is not enabled,
```
since we're not talking about just one feature here